libraries/vault_service.rb

```diff
@@ -76,6 +76,7 @@ def action_enable
 
           execute "setcap cap_ipc_lock=+ep #{new_resource.program}" do
             not_if { platform_family?('windows', 'mac_os_x', 'freebsd') }
+            not_if { platform_family?('rhel') && platform_version.to_i < 6 }
```

Comment:
Travis build has failed because the method `platform_version` is undefined. This is the ohai attribute, so the correct notation is:
```ruby
#...
not_if { platform_family?('rhel') && node['platform_version'].to_i < 6
```
https://docs.chef.io/resource_common.html#not-if-examples